tests/form_pages/shared/test_location_tier.py

```diff
@@ -52,18 +63,29 @@ def test_update_location_tier_by_postcode_should_not_update_when_tiering_logic_d
 def test_update_location_tier_by_postcode_should_update_session_when_tiering_logic_enabled():
     try:
         postcode = "LS11BA"
+        la = "E06000008"
         location_tier = PostcodeTier.MEDIUM.value
+        shielding_advice = YesNoAnswers.YES.value
         _current_app.is_tiering_logic_enabled = True
+        _current_app.shielding_advice = lambda ladcode: shielding_advice
         with _current_app.app_context(), \
-            patch("vulnerable_people_form.form_pages.shared.location_tier.get_postcode_tier",
-                  return_value=location_tier) as mock_get_location_tier, \
-            patch("vulnerable_people_form.form_pages.shared.location_tier.set_location_tier") \
-                as mock_set_location_tier:
+             patch("vulnerable_people_form.form_pages.shared.location_tier.get_postcode_tier",
+                   return_value=location_tier) as mock_get_location_tier, \
+             patch("vulnerable_people_form.form_pages.shared.location_tier.get_ladcode_from_postcode",
+                   return_value=la) as mock_get_ladcode_from_postcode, \
+             patch("vulnerable_people_form.form_pages.shared.location_tier.set_location_tier") \
+                as mock_set_location_tier, \
+             patch("vulnerable_people_form.form_pages.shared.location_tier.set_shielding_advice") \
+                as mock_set_shielding_advice:
+
             update_location_tier_by_postcode(postcode, _current_app)
+            mock_get_ladcode_from_postcode.assert_called_once()
             mock_get_location_tier.assert_called_once_with(postcode)
             mock_set_location_tier.assert_called_once_with(location_tier)
+            mock_set_shielding_advice.assert_called_once_with(shielding_advice)
     finally:
         _current_app.is_tiering_logic_enabled = False
+        del _current_app.shielding_advice
```

Comment:
whats the reasoning for deleting the shielding advice off the conttext?

Reply:
because the previous line does something similar, I'm not sure I really understand why it's needed either.

Reply:
@sanisc I can take it out, I was trying to keep things consistent with what was already there.

Reply:
the previous line is there so that tiering logic is false by default as the current app i think is kept for the lifecycle of all tests. I think the `del _current_app.shielding_advice` can be removed as it is not used in any other tests so its safe to just leave it always on the current app @samcrang